src/protocol/protocol_client.py

```diff
@@ -185,8 +185,8 @@ async def authenticate_token(self, steam_id, account_name, token, auth_lost_hand
         result = await self._login_future
         if result == EResult.OK:
             self._auth_lost_handler = auth_lost_handler
-        elif result == EResult.InvalidPassword:  # until we solve real problem with temporarily invalid token
-            raise galaxy.api.errors.BackendError({"result": result})
+        elif result == EResult.InvalidPassword:
+            raise galaxy.api.errors.InvalidCredentials({"result": result})
         else:
             logger.warning(f"Received unknown error, code: {result}")
             raise translate_error(result)
```

Comment:
`translate_error` already includes this case. result   

```suggestion
        else:
            logger.warning(f"authenticate_token failed with code: {result}")
            raise translate_error(result)
```

Reply:
That's a good point. Committing this change.